src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -9,7 +9,7 @@
 import uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.common.DmnStringValue;
 import uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.common.EventInformation;
 import uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.initiatetask.InitiateEvaluateResponse;
-import uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.warningtask.WarningEvaluateResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.warningtask.WarningResponse;
```

Comment:
Can we add Functional tests for the AC's in the ticket?
1. Given the makeAnApplication event has occurred
And the Cancellation DMN rules have run

2.Given the Warning Sub-process has been triggered 
And the Handle Warning service task has started
When the task in the parent process has attribute "hasWarnings" = No

3.Given the Warning Sub-process has been triggered 
When the task in the parent process has attribute "hasWarnings" = Yes
Then the Handle warning service task will make no change